definitions/npm/ramda_v0.26.x/flow_v0.76.x-/test_ramda_v0.26.x_object.js

```diff
@@ -100,9 +114,21 @@ describe("Object", () => {
 
     it('requires the lens works with the functor provided (object)', () => {
       type Data = { a: string }
+      type NestedData = { a: { b: { c: { d: { e: string } } } } };
       const data: Data = {
         a: 'foo',
       }
+      const nestedData: NestedData = {
```

Comment:
Was this building up to a test case? I don't see it used anywhere.

Reply:
Oh, yes was to test `lensPath`. I'll do the change

Reply:
Updated